ISPP-Eventia/eventus
# Feature/#188 visual improvements

Closes #150 

## changes

- visual adjustments
- added links to the profile page and logout
- moved Router root to the App base component (now header can navigate)

## steps to review

- check the visual improvement
- check out the links on the header
